interfaces/enums/network-colors.ts

```diff
@@ -11,6 +11,8 @@ export enum NetworkColors {
   afrodite = "#29b6af",
   irene = "#29b6af",
   iris = "#29b6af",
+  moonbase = "#29b6af",
+  moonbeam = "#29b6af",
```

Comment:
moonbeam color should not be the same as dev networks, it should match `f6c343` (moonriver)